shortcuts/template.xml

```diff
@@ -1295,6 +1295,9 @@
 
             <property name="widgetPath" tag="property" attribute="name|list" value="DefaultSearch-Netflix">plugin://plugin.video.netflix/directory/search/search/?query=</property>
             <property name="widgetPath2" tag="property" attribute="name|list" value="DefaultSearch-Netflix">/</property>
+			
+			<property name="widgetPath" tag="property" attribute="name|list" value="DefaultSearch-Hulu">plugin://slyguy.hulu/?_=search&query=</property>
```

Comment:
Same deal here as in Disney+. 

Need to use `&amp;` instead of `&` due to being xml. Also, use spaces instead of tabs to ensure consistent identation.